helix-term/src/commands/lsp.rs

```diff
@@ -145,6 +150,97 @@ fn sym_picker(
     .truncate_start(false)
 }
 
+fn diag_picker(
```

Comment:
`diagnostics_picker`

Reply:
It follows the same naming schema as the `sym_picker`:

There's a function `symbol_picker` that calls a function `sym_picker`, the same as the function `diagnostics_picker` is calling `diag_picker`